modules/console/pages/config/security/authentication.adoc

```diff
@@ -55,88 +61,74 @@ flowchart TD
-
-=== Basic authentication
-
-For basic authentication, the user submits a username and password directly to Redpanda Console. These credentials are validated and encapsulated within a JWT, which is then used as follows:
+== Prerequisites
 
-* **Kafka authentication:**
-  The provided credentials are mapped to SASL-SCRAM, ensuring that Kafka operations execute with the correct user identity.
+* You must have at least one superuser in Redpanda.
+* The authentication method used in Redpanda Console must match the configuration of the Kafka API:
 
-* **HTTP API authentication:**
-  The JWT supports HTTP Basic authentication for other APIs.
+**If using OIDC:**
+You must have SASL/OAUTHBEARER authentication configured for the Kafka API.
 
-== Prerequisites
+**If using basic authentication:**
+You must have SASL/SCRAM authentication configured for the Kafka API.
 
-Your credentials must correspond to an existing user in your Redpanda cluster. Redpanda Console verifies access by confirming whether your credentials authenticate with the Kafka API. As a result, the authentication mechanism configured in Redpanda Console must match the Kafka API's configuration. For example:
+xref:manage:security/authentication.adoc[Learn how to configure authentication for Redpanda].
 
-- *OIDC:* If you want to log into Redpanda Console with OIDC, you must have SASL-OAUTHBEARER authentication configured for the Kafka API.
-- *Basic:* If you want to log into Redpanda Console with basic authentication (username/password), you must have SASL-SCRAM authentication configured for the Kafka API.
+== Enable authentication
 
-See xref:manage:security/authentication.adoc[Configure Authentication for Redpanda].
+Redpanda Console supports enabling both OIDC and basic authentication simultaneously. If both are enabled, users can choose how to log in.
 
-[[enable-authentication]]
-== Enable authentication for Redpanda Console
+=== Enable OIDC authentication
 
-Enable OIDC and basic authentication in Redpanda Console. OIDC offers secure single sign-on for production, while basic authentication provides a simple login option.
+To configure OIDC, choose one of two modes:
 
-You can enable both OIDC and basic authentication simultaneously. If you enable both, Redpanda Console presents users with a choice of login methods.
+* *Runtime acquisition mode:* Redpanda Console obtains a token from the IdP.
+* *Static token mode:* You provide a pre-acquired OIDC token.
 
-=== Enable OIDC authentication
+Redpanda and Redpanda Console require OAuth 2.0-compliant JWT tokens for user authentication. When using OIDC, your IdP must issue JWTs. Redpanda Console uses these tokens to authenticate to Redpanda APIs through SASL/OAUTHBEARER or Bearer headers.
 
-To enable OIDC authentication, configure your OIDC settings in the Redpanda Console configuration file. You can obtain an OIDC token using one of the following modes:
+include::manage:partial$security/oidc/limitations.adoc[leveloffset=+2]
```

Comment:
This line in the **Limitations** section seems to imply we will deliver granular authN in the future. Please remove "yet":

"Redpanda requires OIDC principals to be set as superusers to access the Admin API. Granular authorization is not yet supported."